### ISSUE TITLE
Stochastic weight averaging missing

### ISSUE THREAD

**Issue by Ehteshamciitwah** [bharatsubedi/PARseq_torch#6]
Hello, thank you for your pytorch implementation. In the original paper and implementation, the author use Stochastic weight averaging SWA after 75% training and replaced onecycle learning rate scheduler. But in your implementation, you are just using onecycle lr scheduler and claim to achieve the same performance. Is there no effect of SWA on performance? Kindly share your thoughts.
